account_invoice_open_sale/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Open sale order from invoice",
+    "summary": "Show a button on the invoice view to open the "
+               "related sale order",
+    "version": "12.0.1.0.0",
+    "category": "Sales",
+    "website": "htttps://github.com/oca/sale-financial",
+    "author": "Opener B.V., Odoo Community Association (OCA)",
+    "maintainers": ["StefanRijnhart", "MPishpecki"],
```

Comment:
I don't know about you, but I'm not volunteering ;-) I don't believe in this special ownership of a piece of code. I believe in anarchic collaboration instead. Suggest to clear this list.